app/controllers/catalog_controller.rb

```diff
@@ -8,6 +8,8 @@ class CatalogController < ApplicationController
   extend MultilingualLocaleAwareField
 
   before_action do
+    blacklight_config.bulk_actions&.delete_resources!
```

Comment:
overridden here because a) `CatalogController` seems to be the only thing where `blacklight_config.bulk_actions` exists, and b) because the catalog view page is the only place it shows up anyway (right?).